src/GitInfo/build/GitInfo.targets

```diff
@@ -73,6 +73,10 @@
 							in branch, tag or file sources. By default, matches any string that 
 							*ends* in a valid SemVer2 string.
 							Defaults to 'v?(?<MAJOR>\d+)\.(?<MINOR>\d+)\.(?<PATCH>\d+)(?:\-(?<LABEL>[\dA-Za-z\-\.]+))?$|^(?<LABEL>[\dA-Za-z\-\.]+)\-v?(?<MAJOR>\d+)\.(?<MINOR>\d+)\.(?<PATCH>\d+)$'
+	
+	$(GitCommitDateFormat): The value passed as the format option when trying to retrieve
+							the git commit date
+							Defaults to %%cI (windows) or %cI (non windows)

```

Comment:
The two defaults seem to be the same?

Reply:
Windows uses a double percentage. It's the same that was used in the private variables. Keeping the defaults as it was.